tests/link/test_jax.py

```diff
@@ -478,18 +476,18 @@ def assert_fn(x, y):
     compare_jax_and_py(out_fg, [X.astype(config.floatX)], assert_fn=assert_fn)
 
 
-@pytest.mark.xfail(
-    version_parse(jax.__version__) >= version_parse("0.2.12"),
-    reason="Omnistaging cannot be disabled",
-)
-def test_jax_basic_multiout_omni():
-    # Test that a single output of a multi-output `Op` can be used as input to
-    # another `Op`
-    x = dvector()
-    mx, amx = MaxAndArgmax([0])(x)
-    out = mx * amx
-    out_fg = FunctionGraph([x], [out])
-    compare_jax_and_py(out_fg, [np.r_[1, 2]])
+# @pytest.mark.xfail(
+#    version_parse(jax.__version__) >= version_parse("0.2.12"),
+#    reason="Omnistaging cannot be disabled",
+# )
+# def test_jax_basic_multiout_omni():
+#    # Test that a single output of a multi-output `Op` can be used as input to
+#    # another `Op`
+#    x = dvector()
+#    mx, amx = MaxAndArgmax([0])(x)
+#    out = mx * amx
+#    out_fg = FunctionGraph([x], [out])
+#    compare_jax_and_py(out_fg, [np.r_[1, 2]])
```

Comment:
Take a look at how the `MyMultiOut` `Op` is used in `tests.link.test_numba` for an example of what we could do to replace this multi-output `Op` test.